extensions/mssql/src/objectManagement/ui/databaseDialog.ts

```diff
@@ -63,8 +92,15 @@ export class DatabaseDialog extends ObjectManagementDialogBase<Database, Databas
 				], false)
 			};
 
+			// Initilaize Options Tab
+			this.optionsTab = {
+				title: localizedConstants.OptionsSectionHeader,
+				id: 'optionsId',
```

Comment:
nit: The ID is kind of redundant here

```suggestion
				id: 'options',
```